### PR TITLE
Fixes missing language filter

### LINKED ISSUE #10235: Library - Missing 'Language' filter
## Observed behavior
While testing https://github.com/learningequality/kolibri/pull/10233 I noticed that the Language filter in the Library page is no longer being displayed. There's also nothing displayed under the label 'Show resources'.

## Expected behavior
The issue should be further investigated.

## Steps to reproduce the issue
1. Install the the [following build ](https://buildkite.com/learningequality/kolibri-debian/builds/6281) 
2.  Navigate to Learn > Library
3. Observe the the filter section of the page

## Usage Details
Windows 10, Ubuntu 22  - Chrome

### PR DESCRIPTION
**Pull request by akolson**
## Summary

This pr fixes UI regressions with the"Language" filter and "Show resources". Both are now showing as before.
**Before:**
image

**After:**
Screenshot 2023-06-01 at 08 50 22

## References

Fixes https://github.com/learningequality/kolibri/issues/10235

## Reviewer guidance

- Go to `/learn/#/library`
- The language filter and options under "Show resources" should be displayed as before

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by pcenov — APPROVED** [review]
LGTM at @akolson both the Language filter and 'Show resources' are displayed correctly now.

Perhaps just double check whether there should be so much blank space between the activities section and the Language drop-down:

**Comment by akolson** [comment]
~~@jtamiace any thoughts on the spacing? It looks like its been like this for a while now.~~

File: kolibri/plugins/learn/assets/src/views/SearchFiltersPanel/index.vue [hunk 1]
```diff
@@ -371,6 +373,7 @@
 
   .section {
     margin-top: 40px;
+    margin-bottom: 60px;
```
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/views/SearchFiltersPanel/index.vue:379]: Seems like some spacing was added here? This seems like it is causing the extra space that @pcenov commented on?

**Review by rtibbles — COMMENTED** [review]
Code changes are correct - small query about the added spacing extant.

File: kolibri/plugins/learn/assets/src/views/SearchFiltersPanel/index.vue (hunk 1 above)
Comment (akolson) [reply kolibri/plugins/learn/assets/src/views/SearchFiltersPanel/index.vue:379]: Ahh, nice catch. Just realized this class is used allover the search panel. Let me streamline this.

**Comment by akolson** [comment]
@pcenov we should be good with the spacing now.
image

**Review by rtibbles — APPROVED** [review]

**Comment by pcenov** [comment]
Hi @akolson, while regression testing I noticed that now if you don't have the 'Show resources' section then there is not enough padding below the Channel drop-down and it's half hidden. This is valid for all supported browsers:

https://github.com/learningequality/kolibri/assets/79847249/25f6d278-52eb-4b8b-9d27-46c23b8c1b97

**Comment by akolson** [comment]
Hi @pcenov! Let me resolve this more wholesomely. It looks like the initial issue that caused the "Show resources" checkboxes to be hidden also affects other sections.

**Comment by pcenov** [comment]
In that case I'll file this as a separate issue - seems that this particular issue is only valid if there's a scrollbar displayed for the filter's panel and if I am viewing it on a large monitor it's fine.

**Pull request merged**